go/processor/floria/run_context.go

```diff
@@ -76,7 +76,7 @@ func (r runContext) Call(kind tosca.CallKind, parameters tosca.CallParameters) (
 	}
 
 	snapshot := r.CreateSnapshot()
-	if kind != tosca.DelegateCall && kind != tosca.CallCode {
+	if kind != tosca.DelegateCall {
```

Comment:
should this if not check as well for not static call  ?

Reply:
The value should always be 0 in a static call that's why it is not added. I still added it for completeness and to prevent for interpreter implementation errors.